jdi-light/src/main/java/com/epam/jdi/light/settings/WebSettings.java

```diff
@@ -237,39 +285,50 @@ public static void loadCapabilities(String property, String defaultPath, JAction
         } catch (Exception ignore) { }
     }
 
-    private static void setSearchStrategy(String p) {
+    private void setSearchStrategy(String p) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method 'setSearchStrategy(String)' has an NPath complexity of 580](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5482899)